### PR TITLE
MAT-3370 Editor Feature: Save CQL to Browser

### PR DESCRIPTION
**Pull request by adongare**

### CODE REVIEW

**Review by serhii-ilin — DISMISSED** [review]

**Review by serhii-ilin — DISMISSED** [review]

File: src/custom-hooks/useLocalStorage.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from "react";
+
+export const useLocalStorage = (key, defaultValue) => {
```
Comment (jkotanchik-SB) [inline src/custom-hooks/useLocalStorage.ts:3]: Should the parameters have types?

**Review by RohitKandimalla — DISMISSED** [review]

File: src/custom-hooks/useLocalStorage.ts (hunk 1 above)
Comment (adongare) [reply src/custom-hooks/useLocalStorage.ts:3]: I think so. I'll check how hooks work with types.

**Review by gregory-akins — DISMISSED** [review]

File: src/custom-hooks/useLocalStorage.ts (hunk 1 above)
Comment (adongare) [reply src/custom-hooks/useLocalStorage.ts]: added types

**Review by jkotanchik-SB — APPROVED** [review]

File: src/custom-hooks/useLocalStorage.ts (hunk 1 above)
Comment (jkotanchik-SB) [reply src/custom-hooks/useLocalStorage.ts]: Thanks!

**Pull request merged**